tiatoolbox/tools/stainextract.py

```diff
@@ -158,7 +161,8 @@ class MacenkoExtractor:
 
     """
 
-    def __init__(self, luminosity_threshold=0.8, angular_percentile=99):
+    def __init__(self, luminosity_threshold=0.8, angular_percentile=99) -> None:
+        """Initializes :class:`MacenkoExtractor`."""
```

Comment:
```suggestion
        """Initialize :class:`MacenkoExtractor`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090